content/working_with/console/widgets/blueprints.md

```diff
@@ -99,11 +99,12 @@ The default visibility is "Tenant", and according to the logged-in user's permis
 
 1. Click the deploy icon ![Deploy icon]( /images/ui/icons/deploy-icon.png ).   
 2. In the Deploy Blueprint dialog, specify a name for your deployment.
-3. Specify the required deployment inputs.   
+3. Specify environment on which the blueprint should be deployed on - this option is presented only when the blueprint is using [get_environment_capability]({{< relref "developer/blueprints/spec-intrinsic-functions#get_environment_capability" >}}) intrinsic function.
```

Comment:
I wasn't sure if I should mark it at the beginning as `(optional)`
From one point of view this value is not always visible (so it's not required) but form the other side, it's required when presented

I'm open for suggestions regarding that aspect 😄  

If we'd agree to go in a certain direction, I'll also update other files, which are referring to this functionality 🎉

Reply:
I'm fine with this direction.